src/main/resources/application.properties

```diff
@@ -1 +1,13 @@
+# MySQL Connection Properties
+spring.datasource.url=jdbc:mysql://localhost:3306/db_majestic_oman?useSSl=false&serverTimezone=UTC
+spring.datasource.username=root
+spring.datasource.password=salim@123
```

Comment:
We can not expose the password in plaintext in the properties file. For now, we can use an environment variable. Using environment variable will allow us to substitute the password when we run our application